torch/csrc/Module.cpp

```diff
@@ -479,6 +480,14 @@ static PyObject* THModule_rename_privateuse1_backend(
   END_HANDLE_TH_ERRORS
 }
 
+static PyObject* THModule_get_privateuse1_backend_name(
+    PyObject* _unused,
+    PyObject* arg) {
+  HANDLE_TH_ERRORS
+  return THPUtils_packString(c10::get_privateuse1_backend());
+  END_HANDLE_TH_ERRORS
```

Comment:
my main argument against this is that we don't want to add extra API's if we don't need them (since they'll stick around forever, due to BC-risk).

If you're the backend (you're the one manually calling `rename_privateuse1_backend()`), then you should already know what the backend name is. Are you able to use the backend name directly instead of this API?

Reply:
> my main argument against this is that we don't want to add extra API's if we don't need them (since they'll stick around forever, due to BC-risk).
> 
> If you're the backend (you're the one manually calling `rename_privateuse1_backend()`), then you should already know what the backend name is. Are you able to use the backend name directly instead of this API?

yeah, this API is necessary, we have already know the backend name and call the func rename_privateuse1_backend, but in Python with pytorch, we should call this api to get the backend name.